.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,176 @@
+name: Prerelease pipeline
+
+on:
+  release:
+    types:
+      - prereleased
+    tags:
+      - 'v*'
```

Comment:
Won't this trigger a release or otherwise fail in the artifact upload step? Or is it guarded somehow?

Reply:
sorry, I am not sure that I understood question correctly.

Reply:
Actually my bad, for some reason I thought the tags part was under a push but it is not the case. Please ignore it :)